python/antworld/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+include(../../cmake/bob_robotics.cmake)
+BoB_project(SOURCES antworld.cc
+            PYTHON_MODULE antworld
+            BOB_MODULES antworld
+            EXTERNAL_LIBS numpy)
+
+# On gcc I'm getting this warning for static definitions of structs, but it
+# doesn't matter if the remaining fields are filled with zeros, so just suppress
+# it. -- AD
+if(GNU_TYPE_COMPILER)
```

Comment:
A few of the static ones give this warning. The problem is that I want the structs to be const, but I also don't want to have to explicitly set all fields (because they're not all present in all versions of python).

I'm guessing with them being static the members will all just be zeroed anyway, plus examples I found online all seem to do this.